src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));

```

Comment:
`Random `and `ColorSupplier `must be a class field
All magic numbers, strings, and symbols in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names